internal/handlers/common/update_params.go

```diff
@@ -15,132 +15,63 @@
 package common
 
 import (
-	"fmt"
-
 	"go.uber.org/zap"
 
-	"github.com/FerretDB/FerretDB/internal/handlers/commonerrors"
 	"github.com/FerretDB/FerretDB/internal/handlers/commonparams"
 	"github.com/FerretDB/FerretDB/internal/types"
-	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
-// UpdatesParams represents parameters for update command.
+// UpdatesParams represents parameters for the update command.
 type UpdatesParams struct {
-	DB         string
-	Collection string
-	Updates    []UpdateParams
+	DB         string         `ferretdb:"$db"`
+	Collection string         `ferretdb:"collection"`
+	Updates    []UpdateParams `ferretdb:"updates"`
```

Comment:
Should `Updates` be `opt` parameter?

Reply:
No, it’s a required parameter.

Reply:
Ah okie so we need to handle `ErrFieldNotPopulated` for missing field. :hugs:

Reply:
Make it handled inside `checkAllRequiredFIeldsPopulated`.

Reply:
Yeah looks good